packages/cli/src/engine/deployer.ts

```
@@ -218,20 +229,56 @@ export class StripeDeployer {
+      });
+
+      if (result.data.length > 0) {
+        return result.data[0];
+      }
+
+      return null;
+    } catch (error: unknown) {
+      // Search API returns [] when no match found, not an error.
+      // This catch is defensive for actual API errors (auth, network, etc.)
+      throw error;
+    }
```

ESLint

[error] Unnecessary try/catch wrapper.

(no-useless-catch)